drivers/volume/portworx/portworx.go

```diff
@@ -1815,6 +1815,21 @@ func (d *portworx) GetCloudsnaps(volumeName string, params map[string]string) ([
 
 }
 
+// GetCloudsnaps returns all the cloud snaps of the given volume
+func (d *portworx) GetCloudsnapsOfGivenVolume(volumeName string, sourceVolumeID string, params map[string]string) ([]*api.SdkCloudBackupInfo, error) {
+	var token string
+	token = d.getTokenForVolume(volumeName, params)
+	if val, hasKey := params[refreshEndpointParam]; hasKey {
+		refreshEndpoint, _ := strconv.ParseBool(val)
+		d.refreshEndpoint = refreshEndpoint
+	}
+	cloudSnapResponse, err := d.csbackupManager.EnumerateWithFilters(d.getContextWithToken(context.Background(), token), &api.SdkCloudBackupEnumerateWithFiltersRequest{SrcVolumeId: sourceVolumeID})
```

Comment:
Just curious as to whether there is a wrapper function around this in `sched-ops`?

Reply:
Could not find any such method